getting_started/administrators.adoc

```diff
@@ -67,43 +67,23 @@ already listening on those ports you must stop that service before launching
 the {product-title} container.
 ====
 
-*Installing and Starting an All-in-One Server*
+*Starting an All-in-One Server*
 
-. Launch the server in a container:
+. Launch the server:
 +
 ----
-$ sudo docker run -d --name "origin" \
-        --privileged --pid=host --net=host \
-        -v /:/rootfs:ro -v /var/run:/var/run:rw -v /sys:/sys -v /sys/fs/cgroup:/sys/fs/cgroup:rw \
-        -v /var/lib/docker:/var/lib/docker:rw \
-        -v /var/lib/origin/openshift.local.volumes:/var/lib/origin/openshift.local.volumes:rslave \# <1>
-        openshift/origin start
+$ oc cluster up
```

Comment:
Do we need to say something about setting up an insecure registry first? Or maybe just link to this doc:

https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md

Reply:
I wouldn't bother at this point. The output from `oc cluster up` for now should be sufficient. When we will re-shape the current path I'd move the above doc to our official docs, after it gets updated, so that we can link it from here as well.